Formula/xcinfo.rb

```diff
@@ -15,9 +15,8 @@ class Xcinfo < Formula
     sha256 cellar: :any_skip_relocation, catalina:       "4cd605c355e9c18be2219e5b459847673066ada976afee46c49f4b667c91fba5"
   end
 
-  depends_on xcode: ["12.4", :build]
-  depends_on macos: :catalina
-  depends_on :macos
```

Comment:
can you add this back?, and remove line 19 (I believe it would be covered by xcode version constraint)

Reply:
Well, the build needs Xcode 14.

Reply:
So then they should be updated, not removed

Reply:
Do you mean I should re-add
`-  depends_on :macOS`?

Reply:
Done that

Reply:
> You should re-add all of them with the correct information

Can you please guide me what the correct information is?

Reply:
Looks fine now based on your earlier comment.

Reply:
Thanks!